src/Resources/views/Error/error_layout.html.twig

```diff
@@ -0,0 +1,83 @@
+{% spaceless %}
```

Comment:
I think so too, because we're already in the `Error` "namespace" the `error_` prefix is unnecessary.